FESS-GUI/windowsnames.h

```diff
@@ -2,6 +2,12 @@
 #define WINDOWSNAMES
 
 #define MAINWINDOW_TITLE "FlyCAM - Flywheel Control and Monitoring"
+#define MAINWINDOW_ERROR "FlyCAM - Error"
```

Comment:
Just to play devil's advocate, what's wrong with the #defines? Consistency? I actually prefer strings too, but I couldn't find anything online saying which to use when.